test/locationscale.jl

```diff
@@ -44,13 +44,13 @@ function test_location_scale_normal(μ::Float64,σ::Float64,μD::Float64,σD::Fl
 
     insupport(d,0.4) == insupport(dref,0.4)
     @test pdf(d,0.1) ≈ pdf(dref,0.1)
-    @test pdf(d,2:4) ≈ pdf(dref,2:4)
+    @test pdf.(d,2:4) ≈ pdf.(dref,2:4)
     @test logpdf(d,0.4) ≈ logpdf(dref,0.4)
     @test loglikelihood(d,[0.1,0.2,0.3]) ≈ loglikelihood(dref,[0.1,0.2,0.3])
     @test cdf(d,μ-0.4) ≈ cdf(dref,μ-0.4)
     @test logcdf(d,μ-0.4) ≈ logcdf(dref,μ-0.4)
     @test ccdf(d,μ-0.4) ≈ ccdf(dref,μ-0.4) atol=1e-100
-    @test logccdf(d,μ-0.4) ≈ logccdf(dref,μ-0.4) atol=1e-20
+    @test logccdf(d,μ-0.4) ≈ logccdf(dref,μ-0.4) atol=1e-16
```

Comment:
AFAICT the large change of the tolerance is needed on 0.7 because this was ignored on 0.6. The value I get on both 0.6 and 0.7 are `~ -0.34` and the `eps` there is `~1e-17`. An absolute tolerance of `1e-20` here is basically the same as exactly equal.